frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -75,6 +84,26 @@ export const navigationLogic = kea<navigationLogicType<UserType, SystemStatus>>(
                 return user?.team?.id
             },
         ],
+        demoWarning: [
+            () => [userLogic.selectors.user, organizationLogic.selectors.currentOrganization],
+            (user: UserType, organization: OrganizationType): WarningType => {
+                const yesterday = new Date()
+                yesterday.setDate(new Date().getDate() - 1)
+
+                if (Date.parse(organization.created_at) >= yesterday.getTime() && user.team?.is_demo) {
+                    return 'welcome'
```

Comment:
:thinking: We now always show welcome message, even if user has completed setup. Intentional?

Reply:
Not intentional and great catch, fixed!